resources/assets/css/_buttons.css

```diff
@@ -5,10 +5,16 @@
 .button-primary {
     @apply button-generic bg-theme-primary-600 text-white;
 }
+.theme-dark .button-secondary {
```

Comment:
put it near the regular `button-secondary`

Reply:
That's actually a typo, it's meant to be primary. I just re-checked and primary is the same in dark mode so will remove it